js/src/index.ts

```diff
@@ -74,7 +74,7 @@ class AutoversionWidget extends Widget {
     const type = document.createElement("select");
     type.appendChild(default_none);
 
-    request("get",
+    void request("get",
```

Comment:
why does this need a type?

Reply:
Here's what you get without the `void` when you run `make lint`:

```
error  Promises must be handled appropriately or explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises
```

It's due to the updates made to the `recommended-requiring-type-checking` rule preset that I talk about in the comment above: https://github.com/timkpaine/jupyterlab_autoversion/pull/26#issuecomment-699217318